convert2rhel/utils.py

```diff
@@ -382,17 +382,17 @@ def backup(self):
         """ Save current version of a file """
         loggerinst = logging.getLogger(__name__)
         loggerinst.info("Backing up %s" % self.filepath)
-        if (os.path.isfile(self.filepath) and
-                os.path.isdir(TMP_DIR)):
+        if not os.path.isdir(TMP_DIR):
```

Comment:
Maybe check if the dir is writable?  using `os.access('/path/to/folder', os.W_OK)`  W_OK for write and R_OK for read
https://docs.python.org/2.7/library/os.html#os.access

Reply:
Thanks. It actually made me realize the condition for checking the TMP_DIR is not necessary since there's the copying right below and if that raises an IOError (for example for the reason of the TMP_DIR not being accessible) we abort the conversion with printing the reason.